algorithms-modules/algorithms-miscellaneous-7/pom.xml

```diff
@@ -13,4 +13,40 @@
         <version>1.0.0-SNAPSHOT</version>
     </parent>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.openjdk.jmh</groupId>
+            <artifactId>jmh-core</artifactId>
+            <version>1.35</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.openjdk.jmh</groupId>
+            <artifactId>jmh-generator-annprocess</artifactId>
+            <version>1.35</version>
+        </dependency>
+    </dependencies>
+
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
+                    <annotationProcessorPaths>
+                        <path>
+                            <groupId>org.openjdk.jmh</groupId>
+                            <artifactId>jmh-generator-annprocess</artifactId>
+                            <version>1.35</version>
+                        </path>
+                    </annotationProcessorPaths>
+                </configuration>
+            </plugin>
+        </plugins>
```

Comment:
Do we really need to override the plugin configuration? I'm asking because I see the use jmh-generator-annprocess dependency in other maven modules but I don't see the plugin configuration.

Reply:
Removed. Some modules include this configuration. However, it is not required, as Maven will use service discovery to find the proper annotation processor.

Reply:
Removed. Some modules include this configuration. However, it is not required, as Maven will use service discovery to find the proper annotation processor.

Reply:
Thanks for the clarification!